custom_image_utils/shell_script_generator.py

```diff
@@ -62,12 +62,19 @@
   done
 
   echo 'Creating disk.'
+  if [ '{source_image_family_uri}' = '' -o  '{source_image_family_uri}' = 'None' ]; then
```

Comment:
nit: we prefer this syntax `[[ '{source_image_family_uri}' == '' ||  '{source_image_family_uri}' == 'None' ]]`.

Reply:
Updated syntax.